docs-en/chapter_data_structure/number_encoding.md

```diff
@@ -2,11 +2,11 @@
 
 !!! note
 
-    In this book, chapters marked with an * symbol are optional reads. If you are short on time or find them challenging, you may skip these initially and return to them after completing the essential chapters.
+    In this book, chapters marked with an * symbol are optional to read. If you are short on time or find them challenging, you may skip these initially and return to them after completing the essential chapters.
```

Comment:
Hi YangXuanyi
all of these I'm mentioning are optional:
you can change "In this book, chapters marked with an * symbol are optional to read. If you are short on time or find them challenging, you may skip these initially and return to them after completing the essential chapters."
to ->
"In this book, chapters marked with an asterisk '*' are optional readings. If you have limited time or find them challenging to understand, you may initially skip these chapters and tackle them after completing the essential ones."

Reply:
Thank you very much for your suggestion. Among your suggestions, I think this is worth learning from: "with an asterisk '*' are optional readings." I accept your suggestion. The rest is translated as I did before.